stable/mission-control/templates/mission-control-statefulset.yaml

```diff
@@ -148,13 +148,8 @@ spec:
         - name: PGPASSWORD
           valueFrom:
             secretKeyRef:
-              {{- if .Values.postgresql.postgresqlPassword }}
-              name: {{ template "mission-control.fullname" . }}-db-cred
-              key: postgres-password
-            {{- else }}
               name: {{ .Release.Name }}-postgresql
```

Comment:
This won’t work with an external posters server.

Reply:
This condition is only for {{- if .Values.postgresql.enabled }}